pkg/branding/brandedtext.go

```diff
@@ -0,0 +1,15 @@
+package branding
+
+const (
+	productNameRHACS = "Red Hat Advanced Cluster Security for Kubernetes"
+
+	productNameStackrox = "StackRox"
+)
+
+// GetProductName returns the product name based on ProductBranding env variable
+func GetProductName() string {
+	if getProductBrandingEnv() == "RHACS_BRANDING" {
```

Comment:
You already have `ProductBrandingRHACS`
```suggestion
	if getProductBrandingEnv() == ProductBrandingRHACS {
```

Reply:
Done